reverse_replication/README.md

```diff
@@ -65,4 +59,39 @@ The file should be a list of JSONs as:
     "dbName": "db2"
     }
 ]
+```
+
+## Sample Commands
+Checkout out the reverse replication folder from the root:
+```
+cd reverse_replication
+```
+### Quickstart
+Run the launcher command via:
+```sh
+go run launcher.go -projectId=my-project -dataflowRegion=us-east1 -instanceId=my-instance -dbName=mydb -sourceShardsFilePath=gs://bucket-name/shards.json  -sessionFilePath=gs://bucket-name/session.json 
+``` 
+### Custom Names
+Run the launcher command via:
+```sh
+go run launcher.go -projectId=my-project -dataflowRegion=us-east1 -jobNamePrefix=reverse-rep -changeStreamName=mystream -instanceId=my-instance -dbName=mydb -metadataInstance=my-instance -metadataDatabase=stream-metadb -pubSubDataTopicId=my-topic -sourceShardsFilePath=gs://bucket-name/shards.json  -sessionFilePath=gs://bucket-name/session.json 
+``` 
+### Tune Dataflow Configs
+Run the launcher command via:
+```sh
+go run launcher.go -projectId=my-project -dataflowRegion=us-east1 -instanceId=my-instance -dbName=mydb -sourceShardsFilePath=gs://bucket-name/shards.json  -sessionFilePath=gs://bucket-name/session.json -machineType=e2-standard-2 -orderingWorkers=10 -writerWorkers=8
+``` 
+### Run from a specific timestamp
```

Comment:
We should not give this until the backfill issue is fixed. It will only lead to customer frustration.